Tests/PINRemoteImageTests.m

```diff
@@ -60,6 +61,33 @@ - (void)swizzled_scheduleDownloadWithRequest:(NSURLRequest *)request
 @end
 
 #if DEBUG
+
+@interface PINRemoteImageWeakContainer : NSObject
+
+@property (nonatomic, readonly, weak) id target;
+
++ (PINRemoteImageWeakContainer *)weakContainerWithTarget:(id)target;
+- (instancetype)initWithTarget:(id)target;
+
+@end
+
+@implementation PINRemoteImageWeakContainer
+
++ (PINRemoteImageWeakContainer *)weakContainerWithTarget:(id)target
+{
+    return [[self alloc] initWithTarget:target];
+}
+
+- (instancetype)initWithTarget:(id)target
+{
+    if (self) {
```

Comment:
`[super init]`?

Reply:
I'd also not expect to re-define this class in the tests...

Reply:
Why is this class duplicated in the test? I don't think that should happen,